apps/dashboard/app/apps/ood_app.rb

```diff
@@ -264,7 +264,9 @@ def run_setup_production
           #
           # This makes the execution of the setup-production script use the same ruby versions
           # that Passenger uses when launching the app.
-          output, status = Open3.capture2e({'PATH' => path.join('bin').to_s + ':'+ ENV['PATH']}, 'bundle','exec', setup)
+          cmd = [setup]
+          cmd.prepend(%w[bundle exec]) if File.exist?('./Gemfile')
+          output, status = Open3.capture2e({'PATH' => path.join('bin').to_s + ':'+ ENV['PATH']}, cmd)
```

Comment:
Yea, this doesn't work quite right. As is, it errors saying `no implicit conversion of Array into String`.

If we splat it like `*cmd` it gets tripped up. Somehow we need either `'bundle'` as a string and the arguments `['exec', setup]` or just `setup`.